test/e2e/specs/editor/various/preferences.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'preferences', () => {
+	test( 'remembers sidebar dismissal between sessions', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		// Open by default.
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
```

Comment:
The test seems to be failing on this line. 

The issue could be that Playwright isn't waiting for the element. You could try using `expect.poll`.

It'd be good to try changing the classname selector to a role selector. This is what my browser dev tools shows as the accessibility properties:
![Screen Shot 2022-09-15 at 5 05 05 pm](https://user-images.githubusercontent.com/677833/190363361-0cc010af-70dd-4172-98ed-7a3d2b522bd5.png)

(so role is "button", name is "Post (selected)")

Reply:
Hi @talldan, Previously also tried `expect.poll` but expect.poll excepts function only. I also tried to pass the async function within it but still it seems to be not working. 

![image](https://user-images.githubusercontent.com/41000648/190611703-d060a041-b06c-4ac8-8745-bfbb7c9abf42.png)

Let me know your thoughts on same.